python/doc/parquet.rst

```diff
@@ -64,3 +76,16 @@ a Parquet file, set the ``chunk_size`` to a fraction of the total number of rows
 If you also want to compress the columns, you can select a compression
 method using the ``compression`` argument. Typically, ``GZIP`` is the choice if
 you want to minimize size and ``SNAPPY`` for performance.
+
+Instead of writing to a file, you can also write to Python ``bytes`` by
+utilizing an :class:`pyarrow.io.InMemoryOutputStream()`:
+
+.. code:: python
+
+    import pyarrow.io as paio
+    import pyarrow.parquet as pq
+
+    table = ...
+    output = paio.InMemoryOutputStream()
```

Comment:
This is also in the top level namespace, would prefer users get the symbols from there